_data/docs.yml

```
@@ -208,14 +208,14 @@ libs:
   librmm:
     name: librmm
     path: librmm
-    desc: 'RAPIDS Memory Manager (RMM) is a central place for all device memory allocations in cuDF (C++ and Python) and other RAPIDS libraries. In addition, it is a replacement allocator for CUDA Device Memory (and CUDA Managed Memory) and a pool allocator to make CUDA device memory allocation / deallocation faster and asynchronous.'
+    desc: 'RAPIDS Memory Manager (RMM) is a central place for all device memory allocations in cuDF (C++ and Python) and other RAPIDS libraries. In addition, it is a replacement allocator for CUDA Device Memory (and CUDA Managed Memory) and a pool allocator to make CUDA device memory allocation / deallocation faster and asynchronous. Note that as of RAPIDS 23.12 librmm docs are integrated into the Sphinx documentation alongside the rmm Python docs.'
```

```suggestion
    desc: 'RAPIDS Memory Manager (RMM) is a central place for all device memory allocations in cuDF (C++ and Python) and other RAPIDS libraries. In addition, it is a replacement allocator for CUDA Device Memory (and CUDA Managed Memory) and a pool allocator to make CUDA device memory allocation / deallocation faster and asynchronous. Note that as of RAPIDS 23.12 librmm docs are integrated into the Sphinx documentation alongside the RMM Python docs.'
```
Minor: capitalization consistency